src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2626,27 +2615,50 @@ namespace winrt::TerminalApp::implementation
     }
 
     // Method Description:
-    // - Returns a com_ptr to the implementation type of the given tab if it's a TerminalTab.
-    //   If the tab is not a TerminalTab, returns nullptr.
+    // - The TabView does not apply compact sizing to items added after Compact is enabled.
+    //   By forcibly reapplying compact sizing every time we add a new tab, we'll make sure
+    //   that it works.
+    //   Workaround from https://github.com/microsoft/microsoft-ui-xaml/issues/2711
+    //   TODO: Remove this function and its calls when ingesting the above changes.
```

Comment:
Bare TODO

Reply:
This is my todo, you do not need to tag it.